Makefile

```diff
@@ -91,6 +91,12 @@ kpod: .gopathok $(shell hack/find-godeps.sh $(GOPKGDIR) cmd/kpod $(PROJECT))
 crio.conf: crio
 	./crio --config="" config --default > crio.conf
 
+rpm:
+	git archive --format=tar.gz --prefix=cri-o-testonly/ HEAD > cri-o-testonly.tar.gz
```

Comment:
are you planning to lose the '-testonly' at some point?

Reply:
The specfile in here should always be -testonly IMHO. The one without `-testonly` is already at line#3 in the spec file.

Even when we get to delivering rpms automagically from github, those rpmbuild operations should occur on the canonical spec file, and not this one.